arviz/stats/stats.py

```diff
@@ -823,12 +824,16 @@ def summary(
         Refer to documentation of az.convert_to_dataset for details
     var_names : list
         Names of variables to include in summary
-    include_circ : bool
-        Whether to include circular statistics
     fmt : {'wide', 'long', 'xarray'}
         Return format is either pandas.DataFrame {'wide', 'long'} or xarray.Dataset {'xarray'}.
+    kind : {'all', 'stats', 'diagnostics'}
+        Whether to include the `stats` `mean`, `sd`, `hpd_3%`, `hpd_97%`, or the diagnostics
+        `mcse_mean`, `mcse_sd`, `ess_bulk`, `ess_tail`, and `r_hat`. Default to include `all` of
+        them.
```

Comment:
Very minor formatting comment. I would use the same format for stats and for diagnostics, maybe a colon after them too.